test/browser.transforms.test.js

```diff
@@ -547,16 +547,23 @@ describe('scrubPayload', function () {
     expect(payload.data.custom.okay).to.eql('fizz=buzz&fuzz=baz');
     expect(payload.data.custom.user.id).to.eql(42);
 
-    var scrub = require('../src/scrub');
-    t.addScrubber(scrub)(payload, options, function (e, i) {
-      expect(i.access_token).to.eql(accessToken);
-      expect(i.data.custom.scooby).to.not.eql('doo');
-      expect(payload.data.custom.okay).to.not.eql('fizz=buzz&fuzz=baz');
-      expect(payload.data.custom.okay).to.match(/fizz=\*+&fuzz=baz/);
-      expect(payload.data.custom.user.id).to.not.be.ok();
-      expect(payload.data.custom.user).to.match(/\*+/);
-      expect(i.data.message).to.eql('a message');
-      done(e);
+    const scrubModule = await import('../src/scrub.js');
+    const scrub = scrubModule.default;
+
+    const scrubberFn = t.addScrubber(scrub);
+    const result = await new Promise((resolve, reject) => {
+      scrubberFn(payload, options, (err, result) => {
+        if (err) reject(err);
+        else resolve(result);
+      });
```

Comment:
I'm not sure if this is the best way of doing this, in node you can use `promisify`, but I don't know if there's an equivalent available in this environment.

Reply:
https://github.com/mikehall314/es6-promisify as a dev dependency would be ok, but I think this is fine as is.